src/ops/fuelup_check.rs

```diff
@@ -60,18 +60,18 @@ fn check_plugin(plugin_executable: &Path, plugin: &str, latest_version: &Version
             match output.split_whitespace().last() {
                 Some(v) => {
                     let version = Version::parse(v)?;
-                    print!("    - ");
+                    print!("{}- ", pad_spaces(4));
```

Comment:
```suggestion
                    println!("{:>4}- ", "");
```

This seems more idiomatic.